domain/src/test/kotlin/com/studentcenter/weave/domain/user/vo/NicknameTest.kt

```diff
@@ -0,0 +1,28 @@
+package com.studentcenter.weave.domain.user.vo
+
+import io.kotest.assertions.throwables.shouldThrow
+import io.kotest.core.annotation.DisplayName
+import io.kotest.core.spec.style.FunSpec
+import io.kotest.matchers.shouldBe
+
+@DisplayName("NicknameTest")
+class NicknameTest : FunSpec ({
+
+    test("최대 글자수 기준(10) 보다 긴 닉네임을 생성하는 경우, 에러가 발생한다.") {
+
+        val exception = shouldThrow<IllegalArgumentException> {
+            Nickname("닉".repeat(11))
```

Comment:
p4) 최대값이 변경되면 테스트 관리에 리소스가 사용되지 않도록 관리되면 좋을 것 같아요
```suggestion
            Nickname("닉".repeat(Nickname.MAX_LENGTH + 1))
```